### PR TITLE
Refs #35763 - cleanup for switching to PF4 search input

### PR DESCRIPTION
**Pull request by lfu**
#### What are the changes introduced in this pull request?
Remove the usage of [webpack](https://github.com/Katello/katello/tree/master/webpack)/[components](https://github.com/Katello/katello/tree/master/webpack/components)/[Search](https://github.com/Katello/katello/tree/master/webpack/components/Search)/Search.js

#### Considerations taken when implementing this change?
Followup of https://github.com/Katello/katello/pull/10401

#### What are the testing steps for this pull request?
Check all the search field work well in new host details pages and content view components.
Verify CV filter rules add/edit still work for packages and container image tag.
Verify it does not break anything :)

### CODE REVIEW

File: package.json
Comment (jeremylenz) [inline package.json:55]: I think this will need an ok from Packaging. Also, this component seems to be [no longer maintained](https://github.com/reactjs/react-autocomplete).  How painful of an effort would it be to use Patternfly components instead?

Comment (lfu) [reply package.json:55]: Are you referring to `SearchInput`?

Comment (jeremylenz) [reply package.json:55]: I'm referring to the `Autocomplete` component which is imported here from `react-autocomplete`. Can this be done without bringing in any new libraries (e.g., with no changes to `package.json`)?

Comment (jeremylenz) [reply package.json:55]: As for what existing Patternfly component(s) will replace it, I haven't researched that; I'll leave that up to you to tell me if it's doable or not.

**Comment by lfu** [comment]
@jeremylenz Updated.

File: package.json
Comment (lfu) [reply package.json]: Seems Patternfly doesn't have an equivalent component for `react-autocomplete`. 
`SearchInput` is the best without a new react library. 
I would suggect https://github.com/JedWatson/react-select if current implementation with `SearchInput` does not look good.

**Review by jeremylenz — APPROVED** [review]
Great work @lfu!

Autocomplete on filter rule modal works well. As long as tests are passing this LGTM.

APJ :+1:

**Pull request merged**
